### DISCUSSION TITLE
Mark "viewed" files in the File Tree

### DISCUSSION THREAD

**Discussion by OmarTawfik** [community/community#20000] [Pull Requests]
Similar to what the VS Code extension does, by marking viewed files with a green tick (screenshot below), it would be great if GitHub web UI can show the status of these files as well, maybe by changing the icon color, or adding another icon next to it. Helps greatly when reviewing large PRs.

It would also be great if we can mark them as reviewed by clicking the icon/button in the file tree directly, instead of clicking/scrolling to specific files.

## Web UI (no marker):

<img width="313" alt="image" src="https://user-images.githubusercontent.com/15987992/177219046-6cccc5cc-6c20-4903-9144-404660082db3.png">

## VSCode UI (tick marker):

<img width="240" alt="image" src="https://user-images.githubusercontent.com/15987992/177219008-baee58bc-5434-4e1c-b7b5-973e18d6297f.png">

## VSCode UI (mark as "viewed" button):

<img width="218" alt="image" src="https://user-images.githubusercontent.com/15987992/177221062-f85025c5-3148-49fb-b911-5ec1f17b8208.png">

**Comment by LuigiCardamone** [comment]
It was suggested also here: https://github.com/community/community/discussions/18285
